packages/terra-application-links/src/terra-dev-site/doc/example/ApplicationTabsWithIconsExample.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+/* eslint-disable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+import { ApplicationTabs } from 'terra-application-links/lib/ApplicationLinks';
+import ApplicationLinkConfig from 'terra-application-links/lib/terra-dev-site/doc/common/ApplicationLinksWithIconsConfig';
+/* eslint-enable import/no-extraneous-dependencies, import/no-unresolved, import/extensions */
+
+const demoheaderstyles = {
```

Comment:
You'll want to move these styles to a SCSS file and likely want to make the background themable. See this PR for more info: https://github.com/cerner/terra-framework/pull/550